src/OrchardCore.Modules/OrchardCore.Workflows/ResourceManagementOptionsConfiguration.cs

```diff
@@ -47,7 +47,7 @@ static ResourceManagementOptionsConfiguration()
 
         _manifest
             .DefineScript("workflow-editor")
-            .SetDependencies("jsplumb", "bootstrap")
+            .SetDependencies("jsplumb", "admin")
```

Comment:
Here is the equivalent change to have a dependency on theadmin.js
The issue is that the workflow-editor.js file was added before theadmin.js file.